### PR TITLE
Create data.md for D3

### PR DESCRIPTION
**Pull request by qwertyhands**
### Description

Create Codecademy Docs term entry on the .data() method for D3.

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by qwertyhands** [comment]
@ishg-153 I believe this was an issue which I took up, but apologies I didn't state the issue number. I've made the requested changes (hopefully satisfactorily). Thanks

**Comment by ishg-153** [comment]
@qwertyhands I apologize for the mix-up. I couldn't find the corresponding issue earlier, thanks for bringing that to my attention. I will go ahead and review the entry.

**Comment by ishg-153** [comment]
I'm sorry to do this again but could you move it back under the `selection` directory terms? So this would be the new file path:
`content/d3/concepts/selection/terms/data/data.md`

Thank you in advance for understanding.

**Comment by qwertyhands** [comment]
@ishg-153 I've changed the file name as requested

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@qwertyhands I have suggested changes. Please address them and let me know when you are done.

File: content/d3/concepts/selection/terms/data/data.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    // Bind data to list items
+    const items = list.selectAll("li")
+      .data(data)
+      .enter()
+      .append("li")
+      .text(d => d);
+  </script>
+</body>
+</html>
```
[the remark this answers is not in the document]
Comment (qwertyhands) [reply content/d3/concepts/selection/terms/data/data.md]: Hopefully, my other updates are OK. But I'm afraid I can't seem to get the output to show for this. I copied and pasted the above new example code to Notepad, added the line console.log(), and saved it as an html file, which I opened to then inspect, hoping I'd get the output screenshot from this. But there seem to be errors. Please help me with this if you can. Or we can leave it if you think that's best - I chose this issue as it was beginner friendly, but I'm very inexperienced with JavaScript yet alone D3.

**Review by ishg-153 — APPROVED** [review]
@qwertyhands This looks good to go.

**Comment by ishg-153** [comment]
@caupolicandiaz Do you have any more suggestions for this entry? 
After your review can I move straight to merging this?

**Comment by ishg-153** [comment]
Merging this entry.

**Pull request merged**